backend/models/cans.py

```diff
@@ -690,6 +693,26 @@ class CAN(BaseModel):
         "Project", secondary="project_cans", back_populates="cans"
     )
 
+    @property
+    def status(self):
+        if object_session(self) is None:
+            return CANStatus.INACTIVE
```

Comment:
I think it would be OK to throw an exception if `object_session(self) is None` - that will never happen when the app is running normally

Reply:
Removed this code instead of having a specific exception handler. I expect that trying to reference None will just throw the exception as you suggest.